terminal/command.go

```diff
@@ -724,7 +726,11 @@ func printcontext(state *api.DebuggerState) error {
 	}
 	if len(state.CurrentThread.File) == 0 {
 		fmt.Printf("Stopped at: 0x%x\n", state.CurrentThread.PC)
-		fmt.Printf("\033[34m=>\033[0m    no source available\n")
+		if dumbTerminal {
```

Comment:
One thing about the refactor, in the example function signature I gave above, `Term.Println(format string, currentLine bool)`, `format` should just be named `str` or something, it should be a pre-formatted string.